build error

```
==========
== CUDA ==
==========

CUDA Version 12.2.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

WARNING: The NVIDIA Driver was not detected.  GPU functionality will not be available.
   Use the NVIDIA Container Toolkit to start this container with GPU support; see
   https://docs.nvidia.com/datacenter/cloud-native/ .

Get:1 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [128 kB]
Get:3 https://developer.download.nvidia.cn/compute/cuda/repos/ubuntu2004/x86_64  InRelease [1581 B]
Get:4 https://developer.download.nvidia.cn/compute/cuda/repos/ubuntu2004/x86_64  Packages [2686 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-updates InRelease [128 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal-backports InRelease [128 kB]
Get:7 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [4801 kB]
Get:8 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
Get:9 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]
Get:11 http://archive.ubuntu.com/ubuntu focal/multiverse amd64 Packages [177 kB]
Get:12 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [4998 kB]
Get:13 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1308 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1599 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [36.8 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [4919 kB]
Get:17 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [4431 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]
Get:19 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]
Get:20 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [33.1 kB]
Fetched 38.4 MB in 14s (2735 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  cmake-data git-man krb5-locales less libarchive13 libbrotli1 libbsd0 libcbor0.6 libcurl3-gnutls libcurl4 libedit2 liberror-perl libexpat1 libfido2-1 libgssapi-krb5-2 libicu66 libjsoncpp1 libk5crypto3
  libkeyutils1 libkrb5-3 libkrb5support0 libnghttp2-14 libpsl5 librhash0 librtmp1 libssh-4 libuv1 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxml2 libxmuu1 openssh-client publicsuffix tzdata
  xauth
Suggested packages:
  cmake-doc ninja-build gettext-base git-daemon-run | git-daemon-sysvinit git-doc git-el git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn lrzip krb5-doc krb5-user keychain libpam-ssh monkeysphere
  ssh-askpass
The following NEW packages will be installed:
  cmake cmake-data git git-man krb5-locales less libarchive13 libbrotli1 libbsd0 libcbor0.6 libcurl3-gnutls libcurl4 libedit2 liberror-perl libexpat1 libfido2-1 libgssapi-krb5-2 libicu66 libjsoncpp1
  libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 libnghttp2-14 libpsl5 librhash0 librtmp1 libssh-4 libuv1 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxml2 libxmuu1 openssh-client
  publicsuffix tzdata xauth
0 upgraded, 41 newly installed, 0 to remove and 82 not upgraded.
Need to get 24.5 MB of archives.
After this operation, 123 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 less amd64 551-1ubuntu0.3 [123 kB]
Get:2 http://archive.ubuntu.com/ubuntu focal/main amd64 libbsd0 amd64 0.10.0-1 [45.4 kB]
Get:3 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libexpat1 amd64 2.2.9-1ubuntu0.8 [75.6 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 tzdata all 2025b-0ubuntu0.20.04.1 [300 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libicu66 amd64 66.1-2ubuntu2.1 [8515 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libxml2 amd64 2.9.10+dfsg-5ubuntu0.20.04.10 [640 kB]
Get:7 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 krb5-locales all 1.17-6ubuntu4.11 [12.0 kB]
Get:8 http://archive.ubuntu.com/ubuntu focal/main amd64 libcbor0.6 amd64 0.6.0-0ubuntu1 [21.1 kB]
Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 libedit2 amd64 3.1-20191231-1 [87.0 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal/main amd64 libfido2-1 amd64 1.3.1-1ubuntu2 [47.9 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libkrb5support0 amd64 1.17-6ubuntu4.11 [31.5 kB]
Get:12 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libk5crypto3 amd64 1.17-6ubuntu4.11 [80.3 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libkeyutils1 amd64 1.6-6ubuntu1.1 [10.3 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libkrb5-3 amd64 1.17-6ubuntu4.11 [330 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libgssapi-krb5-2 amd64 1.17-6ubuntu4.11 [121 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal/main amd64 libpsl5 amd64 0.21.0-1ubuntu1 [51.5 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libuv1 amd64 1.34.2-1ubuntu1.5 [80.9 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal/main amd64 libxau6 amd64 1:1.0.9-0ubuntu1 [7488 B]
Get:19 http://archive.ubuntu.com/ubuntu focal/main amd64 libxdmcp6 amd64 1:1.1.3-0ubuntu1 [10.6 kB]
Get:20 http://archive.ubuntu.com/ubuntu focal/main amd64 libxcb1 amd64 1.14-2 [44.7 kB]
Get:21 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libx11-data all 2:1.6.9-2ubuntu1.6 [114 kB]
Get:22 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libx11-6 amd64 2:1.6.9-2ubuntu1.6 [577 kB]
Get:23 http://archive.ubuntu.com/ubuntu focal/main amd64 libxext6 amd64 2:1.3.4-0ubuntu1 [29.1 kB]
Get:24 http://archive.ubuntu.com/ubuntu focal/main amd64 libxmuu1 amd64 2:1.1.3-0ubuntu1 [9728 B]
Get:25 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 openssh-client amd64 1:8.2p1-4ubuntu0.13 [670 kB]
Get:26 http://archive.ubuntu.com/ubuntu focal/main amd64 publicsuffix all 20200303.0012-1 [111 kB]
Get:27 http://archive.ubuntu.com/ubuntu focal/main amd64 xauth amd64 1:1.1-0ubuntu1 [25.0 kB]
Get:28 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 cmake-data all 3.16.3-1ubuntu1.20.04.1 [1613 kB]
Get:29 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libarchive13 amd64 3.4.0-2ubuntu1.5 [327 kB]
Get:30 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libbrotli1 amd64 1.0.7-6ubuntu0.1 [267 kB]
Get:31 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libnghttp2-14 amd64 1.40.0-1ubuntu0.3 [79.9 kB]
Get:32 http://archive.ubuntu.com/ubuntu focal/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d.1-2build1 [54.9 kB]
Get:33 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libssh-4 amd64 0.9.3-2ubuntu2.5 [171 kB]
Get:34 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libcurl4 amd64 7.68.0-1ubuntu2.25 [235 kB]
Get:35 http://archive.ubuntu.com/ubuntu focal/main amd64 libjsoncpp1 amd64 1.7.4-3.1ubuntu2 [75.6 kB]
Get:36 http://archive.ubuntu.com/ubuntu focal/main amd64 librhash0 amd64 1.3.9-1 [113 kB]
Get:37 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 cmake amd64 3.16.3-1ubuntu1.20.04.1 [3668 kB]
Get:38 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libcurl3-gnutls amd64 7.68.0-1ubuntu2.25 [233 kB]
Get:39 http://archive.ubuntu.com/ubuntu focal/main amd64 liberror-perl all 0.17029-1 [26.5 kB]
Get:40 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 git-man all 1:2.25.1-1ubuntu3.14 [887 kB]
Get:41 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 git amd64 1:2.25.1-1ubuntu3.14 [4613 kB]
Fetched 24.5 MB in 8s (2891 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package less.
(Reading database ... 14133 files and directories currently installed.)
Preparing to unpack .../00-less_551-1ubuntu0.3_amd64.deb ...
Unpacking less (551-1ubuntu0.3) ...
Selecting previously unselected package libbsd0:amd64.
Preparing to unpack .../01-libbsd0_0.10.0-1_amd64.deb ...
Unpacking libbsd0:amd64 (0.10.0-1) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../02-libexpat1_2.2.9-1ubuntu0.8_amd64.deb ...
Unpacking libexpat1:amd64 (2.2.9-1ubuntu0.8) ...
Selecting previously unselected package tzdata.
Preparing to unpack .../03-tzdata_2025b-0ubuntu0.20.04.1_all.deb ...
Unpacking tzdata (2025b-0ubuntu0.20.04.1) ...
Selecting previously unselected package libicu66:amd64.
Preparing to unpack .../04-libicu66_66.1-2ubuntu2.1_amd64.deb ...
Unpacking libicu66:amd64 (66.1-2ubuntu2.1) ...
Selecting previously unselected package libxml2:amd64.
Preparing to unpack .../05-libxml2_2.9.10+dfsg-5ubuntu0.20.04.10_amd64.deb ...
Unpacking libxml2:amd64 (2.9.10+dfsg-5ubuntu0.20.04.10) ...
Selecting previously unselected package krb5-locales.
Preparing to unpack .../06-krb5-locales_1.17-6ubuntu4.11_all.deb ...
Unpacking krb5-locales (1.17-6ubuntu4.11) ...
Selecting previously unselected package libcbor0.6:amd64.
Preparing to unpack .../07-libcbor0.6_0.6.0-0ubuntu1_amd64.deb ...
Unpacking libcbor0.6:amd64 (0.6.0-0ubuntu1) ...
Selecting previously unselected package libedit2:amd64.
Preparing to unpack .../08-libedit2_3.1-20191231-1_amd64.deb ...
Unpacking libedit2:amd64 (3.1-20191231-1) ...
Selecting previously unselected package libfido2-1:amd64.
Preparing to unpack .../09-libfido2-1_1.3.1-1ubuntu2_amd64.deb ...
Unpacking libfido2-1:amd64 (1.3.1-1ubuntu2) ...
Selecting previously unselected package libkrb5support0:amd64.
Preparing to unpack .../10-libkrb5support0_1.17-6ubuntu4.11_amd64.deb ...
Unpacking libkrb5support0:amd64 (1.17-6ubuntu4.11) ...
Selecting previously unselected package libk5crypto3:amd64.
Preparing to unpack .../11-libk5crypto3_1.17-6ubuntu4.11_amd64.deb ...
Unpacking libk5crypto3:amd64 (1.17-6ubuntu4.11) ...
Selecting previously unselected package libkeyutils1:amd64.
Preparing to unpack .../12-libkeyutils1_1.6-6ubuntu1.1_amd64.deb ...
Unpacking libkeyutils1:amd64 (1.6-6ubuntu1.1) ...
Selecting previously unselected package libkrb5-3:amd64.
Preparing to unpack .../13-libkrb5-3_1.17-6ubuntu4.11_amd64.deb ...
Unpacking libkrb5-3:amd64 (1.17-6ubuntu4.11) ...
Selecting previously unselected package libgssapi-krb5-2:amd64.
Preparing to unpack .../14-libgssapi-krb5-2_1.17-6ubuntu4.11_amd64.deb ...
Unpacking libgssapi-krb5-2:amd64 (1.17-6ubuntu4.11) ...
Selecting previously unselected package libpsl5:amd64.
Preparing to unpack .../15-libpsl5_0.21.0-1ubuntu1_amd64.deb ...
Unpacking libpsl5:amd64 (0.21.0-1ubuntu1) ...
Selecting previously unselected package libuv1:amd64.
Preparing to unpack .../16-libuv1_1.34.2-1ubuntu1.5_amd64.deb ...
Unpacking libuv1:amd64 (1.34.2-1ubuntu1.5) ...
Selecting previously unselected package libxau6:amd64.
Preparing to unpack .../17-libxau6_1%3a1.0.9-0ubuntu1_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.9-0ubuntu1) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../18-libxdmcp6_1%3a1.1.3-0ubuntu1_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.3-0ubuntu1) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../19-libxcb1_1.14-2_amd64.deb ...
Unpacking libxcb1:amd64 (1.14-2) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../20-libx11-data_2%3a1.6.9-2ubuntu1.6_all.deb ...
Unpacking libx11-data (2:1.6.9-2ubuntu1.6) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../21-libx11-6_2%3a1.6.9-2ubuntu1.6_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.6.9-2ubuntu1.6) ...
Selecting previously unselected package libxext6:amd64.
Preparing to unpack .../22-libxext6_2%3a1.3.4-0ubuntu1_amd64.deb ...
Unpacking libxext6:amd64 (2:1.3.4-0ubuntu1) ...
Selecting previously unselected package libxmuu1:amd64.
Preparing to unpack .../23-libxmuu1_2%3a1.1.3-0ubuntu1_amd64.deb ...
Unpacking libxmuu1:amd64 (2:1.1.3-0ubuntu1) ...
Selecting previously unselected package openssh-client.
Preparing to unpack .../24-openssh-client_1%3a8.2p1-4ubuntu0.13_amd64.deb ...
Unpacking openssh-client (1:8.2p1-4ubuntu0.13) ...
Selecting previously unselected package publicsuffix.
Preparing to unpack .../25-publicsuffix_20200303.0012-1_all.deb ...
Unpacking publicsuffix (20200303.0012-1) ...
Selecting previously unselected package xauth.
Preparing to unpack .../26-xauth_1%3a1.1-0ubuntu1_amd64.deb ...
Unpacking xauth (1:1.1-0ubuntu1) ...
Selecting previously unselected package cmake-data.
Preparing to unpack .../27-cmake-data_3.16.3-1ubuntu1.20.04.1_all.deb ...
Unpacking cmake-data (3.16.3-1ubuntu1.20.04.1) ...
Selecting previously unselected package libarchive13:amd64.
Preparing to unpack .../28-libarchive13_3.4.0-2ubuntu1.5_amd64.deb ...
Unpacking libarchive13:amd64 (3.4.0-2ubuntu1.5) ...
Selecting previously unselected package libbrotli1:amd64.
Preparing to unpack .../29-libbrotli1_1.0.7-6ubuntu0.1_amd64.deb ...
Unpacking libbrotli1:amd64 (1.0.7-6ubuntu0.1) ...
Selecting previously unselected package libnghttp2-14:amd64.
Preparing to unpack .../30-libnghttp2-14_1.40.0-1ubuntu0.3_amd64.deb ...
Unpacking libnghttp2-14:amd64 (1.40.0-1ubuntu0.3) ...
Selecting previously unselected package librtmp1:amd64.
Preparing to unpack .../31-librtmp1_2.4+20151223.gitfa8646d.1-2build1_amd64.deb ...
Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2build1) ...
Selecting previously unselected package libssh-4:amd64.
Preparing to unpack .../32-libssh-4_0.9.3-2ubuntu2.5_amd64.deb ...
Unpacking libssh-4:amd64 (0.9.3-2ubuntu2.5) ...
Selecting previously unselected package libcurl4:amd64.
Preparing to unpack .../33-libcurl4_7.68.0-1ubuntu2.25_amd64.deb ...
Unpacking libcurl4:amd64 (7.68.0-1ubuntu2.25) ...
Selecting previously unselected package libjsoncpp1:amd64.
Preparing to unpack .../34-libjsoncpp1_1.7.4-3.1ubuntu2_amd64.deb ...
Unpacking libjsoncpp1:amd64 (1.7.4-3.1ubuntu2) ...
Selecting previously unselected package librhash0:amd64.
Preparing to unpack .../35-librhash0_1.3.9-1_amd64.deb ...
Unpacking librhash0:amd64 (1.3.9-1) ...
Selecting previously unselected package cmake.
Preparing to unpack .../36-cmake_3.16.3-1ubuntu1.20.04.1_amd64.deb ...
Unpacking cmake (3.16.3-1ubuntu1.20.04.1) ...
Selecting previously unselected package libcurl3-gnutls:amd64.
Preparing to unpack .../37-libcurl3-gnutls_7.68.0-1ubuntu2.25_amd64.deb ...
Unpacking libcurl3-gnutls:amd64 (7.68.0-1ubuntu2.25) ...
Selecting previously unselected package liberror-perl.
Preparing to unpack .../38-liberror-perl_0.17029-1_all.deb ...
Unpacking liberror-perl (0.17029-1) ...
Selecting previously unselected package git-man.
Preparing to unpack .../39-git-man_1%3a2.25.1-1ubuntu3.14_all.deb ...
Unpacking git-man (1:2.25.1-1ubuntu3.14) ...
Selecting previously unselected package git.
Preparing to unpack .../40-git_1%3a2.25.1-1ubuntu3.14_amd64.deb ...
Unpacking git (1:2.25.1-1ubuntu3.14) ...
Setting up libexpat1:amd64 (2.2.9-1ubuntu0.8) ...
Setting up libxau6:amd64 (1:1.0.9-0ubuntu1) ...
Setting up libkeyutils1:amd64 (1.6-6ubuntu1.1) ...
Setting up libpsl5:amd64 (0.21.0-1ubuntu1) ...
Setting up libbrotli1:amd64 (1.0.7-6ubuntu0.1) ...
Setting up libnghttp2-14:amd64 (1.40.0-1ubuntu0.3) ...
Setting up less (551-1ubuntu0.3) ...
Setting up krb5-locales (1.17-6ubuntu4.11) ...
Setting up libcbor0.6:amd64 (0.6.0-0ubuntu1) ...
Setting up libkrb5support0:amd64 (1.17-6ubuntu4.11) ...
Setting up tzdata (2025b-0ubuntu0.20.04.1) ...

Current default time zone: 'Etc/UTC'
Local time is now:      Sat Jun 28 07:09:17 UTC 2025.
Universal Time is now:  Sat Jun 28 07:09:17 UTC 2025.
Run 'dpkg-reconfigure tzdata' if you wish to change it.

Setting up liberror-perl (0.17029-1) ...
Setting up libuv1:amd64 (1.34.2-1ubuntu1.5) ...
Setting up libx11-data (2:1.6.9-2ubuntu1.6) ...
Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2build1) ...
Setting up libk5crypto3:amd64 (1.17-6ubuntu4.11) ...
Setting up librhash0:amd64 (1.3.9-1) ...
Setting up git-man (1:2.25.1-1ubuntu3.14) ...
Setting up cmake-data (3.16.3-1ubuntu1.20.04.1) ...
Setting up libkrb5-3:amd64 (1.17-6ubuntu4.11) ...
Setting up libfido2-1:amd64 (1.3.1-1ubuntu2) ...
Setting up libbsd0:amd64 (0.10.0-1) ...
Setting up publicsuffix (20200303.0012-1) ...
Setting up libjsoncpp1:amd64 (1.7.4-3.1ubuntu2) ...
Setting up libxdmcp6:amd64 (1:1.1.3-0ubuntu1) ...
Setting up libxcb1:amd64 (1.14-2) ...
Setting up libicu66:amd64 (66.1-2ubuntu2.1) ...
Setting up libedit2:amd64 (3.1-20191231-1) ...
Setting up libgssapi-krb5-2:amd64 (1.17-6ubuntu4.11) ...
Setting up libssh-4:amd64 (0.9.3-2ubuntu2.5) ...
Setting up libcurl4:amd64 (7.68.0-1ubuntu2.25) ...
Setting up libx11-6:amd64 (2:1.6.9-2ubuntu1.6) ...
Setting up libxml2:amd64 (2.9.10+dfsg-5ubuntu0.20.04.10) ...
Setting up libxmuu1:amd64 (2:1.1.3-0ubuntu1) ...
Setting up libarchive13:amd64 (3.4.0-2ubuntu1.5) ...
Setting up openssh-client (1:8.2p1-4ubuntu0.13) ...
Setting up libxext6:amd64 (2:1.3.4-0ubuntu1) ...
Setting up libcurl3-gnutls:amd64 (7.68.0-1ubuntu2.25) ...
Setting up git (1:2.25.1-1ubuntu3.14) ...
Setting up xauth (1:1.1-0ubuntu1) ...
Setting up cmake (3.16.3-1ubuntu1.20.04.1) ...
Processing triggers for libc-bin (2.31-0ubuntu9.12) ...
CI_COMMIT_BRANCH:unknown
CI_COMMIT_SHA:7ea304c9b500054615167f73afe30241808002aa
-- Git hash is 7ea304c
-- Configuring done
-- Generating done
-- Build files have been written to: /libvgpu/build
Scanning dependencies of target nvml_mod
Scanning dependencies of target multiprocess_mod
Scanning dependencies of target cuda_mod
Scanning dependencies of target allocator_mod
[  4%] Built target test_alloc
[  8%] Built target test_runtime_launch
[ 12%] Built target test_alloc_host
[ 19%] Built target test_runtime_alloc_host
[ 19%] Building C object src/allocator/CMakeFiles/allocator_mod.dir/allocator.c.o
[ 21%] Building C object src/nvml/CMakeFiles/nvml_mod.dir/hook.c.o
[ 25%] Built target test_alloc_hold
[ 29%] Built target test_alloc_managed
[ 31%] Building C object src/multiprocess/CMakeFiles/multiprocess_mod.dir/multiprocess_memory_limit.c.o
[ 36%] Built target test_create_3d_array
[ 40%] Built target test_host_alloc
[ 42%] Building C object src/cuda/CMakeFiles/cuda_mod.dir/context.c.o
[ 46%] Built target test_alloc_pitch
[ 51%] Built target test_create_array
[ 53%] Building C object src/multiprocess/CMakeFiles/multiprocess_mod.dir/multiprocess_utilization_watcher.c.o
[ 57%] Built target test_host_register
/libvgpu/src/nvml/hook.c: In function 'nvmlDeviceGetMemoryInfo_v2':
/libvgpu/src/nvml/hook.c:373:44: warning: passing argument 2 of '_nvmlDeviceGetMemoryInfo' from incompatible pointer type [-Wincompatible-pointer-types]
  373 |     return _nvmlDeviceGetMemoryInfo(device,memory,2);
      |                                            ^~~~~~
      |                                            |
      |                                            nvmlMemory_v2_t * {aka struct nvmlMemory_v2_st *}
/libvgpu/src/nvml/hook.c:319:73: note: expected 'nvmlMemory_t *' {aka 'struct nvmlMemory_st *'} but argument is of type 'nvmlMemory_v2_t *' {aka 'struct nvmlMemory_v2_st *'}
  319 | nvmlReturn_t _nvmlDeviceGetMemoryInfo(nvmlDevice_t device,nvmlMemory_t* memory,int version) {
      |                                                           ~~~~~~~~~~~~~~^~~~~~
[ 61%] Built target test_runtime_alloc
/libvgpu/src/multiprocess/multiprocess_memory_limit.c: In function 'init_device_info':
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:984: error: unterminated argument list invoking macro "CHECK_NVML_API"
  984 | }
      |
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:129:9: error: 'CHECK_NVML_API' undeclared (first use in this function)
  129 |         CHECK_NVML_API(nvmlDeviceGetUUID(dev,region_info.shared_region->uuids[i],NVML_DEVICE_UUID_V2_BUFFER_SIZE);
      |         ^~~~~~~~~~~~~~
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:129:9: note: each undeclared identifier is reported only once for each function it appears in
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:129:23: error: expected ';' at end of input
  129 |         CHECK_NVML_API(nvmlDeviceGetUUID(dev,region_info.shared_region->uuids[i],NVML_DEVICE_UUID_V2_BUFFER_SIZE);
      |                       ^
      |                       ;
......
  984 | }
      |
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:129:9: error: expected declaration or statement at end of input
  129 |         CHECK_NVML_API(nvmlDeviceGetUUID(dev,region_info.shared_region->uuids[i],NVML_DEVICE_UUID_V2_BUFFER_SIZE);
      |         ^~~~~~~~~~~~~~
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:129:9: error: expected declaration or statement at end of input
/libvgpu/src/multiprocess/multiprocess_memory_limit.c:984: warning: control reaches end of non-void function [-Wreturn-type]
  984 | }
      |
make[2]: *** [src/multiprocess/CMakeFiles/multiprocess_mod.dir/build.make:63: src/multiprocess/CMakeFiles/multiprocess_mod.dir/multiprocess_memory_limit.c.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 65%] Built target test_runtime_host_register
[ 70%] Built target allocator_mod
[ 70%] Built target test_runtime_alloc_managed
[ 74%] Building C object src/cuda/CMakeFiles/cuda_mod.dir/device.c.o
[ 74%] Built target nvml_mod
[ 74%] Built target python_test
[ 76%] Building C object src/cuda/CMakeFiles/cuda_mod.dir/hook.c.o
[ 80%] Built target test_runtime_host_alloc
[ 82%] Building C object src/cuda/CMakeFiles/cuda_mod.dir/memory.c.o
make[1]: *** [CMakeFiles/Makefile2:287: src/multiprocess/CMakeFiles/multiprocess_mod.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 89%] Built target cuda_mod
make: *** [Makefile:84: all] Error 2
make: *** [build-in-docker] Error 2
```

--- a/src/multiprocess/multiprocess_memory_limit.c
+++ b/src/multiprocess/multiprocess_memory_limit.c
@@ -126,7 +126,7 @@ int init_device_info() {
     nvmlDevice_t dev;
     for(i=0;i<nvmlDevicesCount;i++){
         CHECK_NVML_API(nvmlDeviceGetHandleByIndex(i, &dev));
-        CHECK_NVML_API(nvmlDeviceGetUUID(dev,region_info.shared_region->uuids[i],NVML_DEVICE_UUID_V2_BUFFER_SIZE);
+        CHECK_NVML_API(nvmlDeviceGetUUID(dev,region_info.shared_region->uuids[i],NVML_DEVICE_UUID_V2_BUFFER_SIZE));
     }
     LOG_INFO("put_device_info finished %d",nvmlDevicesCount);
     return 0;

